src/Aspire.Hosting.Dapr/DaprDistributedApplicationLifecycleHook.cs

```diff
@@ -164,13 +156,13 @@ public async Task BeforeStartAsync(DistributedApplicationModel appModel, Cancell
                 throw new DistributedApplicationException("AppId is required for Dapr sidecar executable.");
             }
 
-            var resource = new ExecutableResource(appId, fileName, workingDirectory, daprCommandLine.Arguments.ToArray());
+            var daprSideCar = new ExecutableResource($"{resource.Name}-dapr", fileName, appHostDirectory, daprCommandLine.Arguments.ToArray());
```

Comment:
I wonder if it'd make sense to add an option to allow setting the resource name in case `{resource}-dapr` isn't desired.